latest/index.bs

```diff
@@ -207,8 +207,52 @@ keys as specified below for discovering certain types of data, especially images
 
 Metadata about the multiple resolution representations of the image can be
 found under the "multiscales" key in the group-level metadata.
-The specification for the multiscale (i.e. "resolution") metadata is provided
-in [zarr-specs#50](https://github.com/zarr-developers/zarr-specs/issues/50).
+"multiscales" contains a list of dictionaries where each entry describes a multiscale image.
+
+"multiscales" MUST contain the field "datasets", which is a list of dictionaries describing
+the arrays storing the individual resolution levels. Each dictionary in "datasets" MUST
+contain the field "path", whose value contains the path to the array for this resolution relative
+to the root zarr group. The "path"s MUST be ordered from largest (i.e. highest resolution) to smallest.
+
+It SHOULD contain the field "name".
+
+It SHOULD contain the field "version", which indicates the version of the
+multiscale metadata of this image (current version is 0.1).
+
+It SHOULD contain the field "axes", which is a list of dimension names of the axes.
```

Comment:
What is the default behavior if `axes` is not specified? In particular should the consumer have an expectation with regards to the number of underlying dimensions or try to detect it?

Alternatively, the lifting of the 5D requirement could lead to increasing the requirement for `axes` at a `MUST` level.

Reply:
Yes, maybe it's better to change `axes` to `MUST`. I went for SHOULD to stay compatible with 0.1; but if hierarchical storage becomes `MUST` (see #29) 0.1 will be deprecated anyway and we can also promote `axes` to `MUST`.

Reply:
Any thoughts @joshmoore?

Reply:
No immediate objection to breaking which this loosening of the 5D/strict-ordering set up, but it might be nice as a part of this effort to plan out the next 1 or 2 large changes based on #35 and see which are _also_ going to be breaking.

Reply:
> No immediate objection to breaking now along with this loosening of the 5D/strict-ordering version, but it might be nice as a part of this effort to plan out the next 1 or 2 large changes based on #35 and see which are _also_ going to be breaking.

Good point. I will try to summarise the current discussion in there later so we can see what else down the line might be breaking.

Reply:
I changed to MUST now, also summarised potential changes: https://github.com/ome/ngff/issues/35#issuecomment-800662190. As far as I can see none of this would be breaking with the current proposal for 0.2.